Artesian/__init__.py

```diff
@@ -0,0 +1,9 @@
+from ._Services.QueryService import QueryService
+from ._Query.Config.Granularity import Granularity
+from ._Query.Config.RelativeInterval import RelativeInterval
+from ._Configuration.ArtesianConfig import ArtesianConfig
+from ._Configuration.ArtesianPolicyConfig import ArtesianPolicyConfig
+from ._Services.MarketDataService import MarketDataService
+from ._Services.GMEPublicOfferService import GMEPublicOfferService
+
+__all__ = ["QueryService", "Granularity", "RelativeInterval", "ArtesianConfig", "ArtesianPolicyConfig", "MarketDataService","GMEPublicOfferService"]
```

Comment:
I think here all public Enums of the queries are missing. All public elements should be exposed here as in any usage of this SDK, the client should not have to import from paths with '_' into it.

Reply:
for example GMEPO enums are missing